apprentice_teacher.rb

```diff
@@ -1,40 +1,17 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'teacher'
+
+class ApprenticeTeacher < Teacher
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
   end
 
   def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works. "
-    response += "*drops crazy knowledge bomb* "
-    response += "... You're welcome."
+    response = "Listen, class, this is how everything works. *drops crazy knowledge bomb* ... You're welcome."
     response
   end
 
```

Comment:
teach_stuff method is common in both the Teacher and ApprenticeTeacher classes except for a few words. Can you think of a way to extract it into the Parent class?